### DISCUSSION TITLE
CPU color changing

### DISCUSSION THREAD

**Discussion by Gloobyy** [ixjf/MSIRGB#219] [Q&A]
https://github.com/nagisa/msi-rgb/issues/13

is this possible for MSIRGB? 

it supposedly works on msi-rgb (linux)

**Comment by ixjf** [comment]
Yes. MSIRGB has built-in support to query AIDA64 for temperature values of CPU, GPU, HDD and motherboard, if AIDA64 is installed. Support for querying temperature values reported by other programs can be programmed in using Lua modules.

You can check the wiki for writing Lua scripts for details about AIDA64: https://github.com/ixjf/MSIRGB/wiki/Scripts#aida64-module
